source/usage-examples/findOne.txt

```diff
@@ -10,23 +10,21 @@ for a single document in a collection with ``collection.findOne()``.
 The ``findOne()`` method uses a query document that you provide to match
 only the subset of the documents in the collection that match the query.
 If you don't provide a query document (or if you provide an empty document),
-MongoDB matches all documents in the collection. However, only the first
-matched document is returned.
-
+MongoDB matches all documents in the collection. However, the operation
+only returns the first matched document.
 
 You can also define additional query options such as
 :doc:`sort </fundamentals/crud/read-operations/sort>`
 and :doc:`projection </fundamentals/crud/read-operations/project>`
-to configure the returned document. You should specify these options
-through the ``options`` object passed as the second parameter of the
-``findOne`` method. For detailed reference documentation, see
+to configure the returned document. Pass them in the ``options`` object as the
```

Comment:
I think this revision is actually a regression. The new sentence begins with a phrase that doesn't sound right to me. "Pass them in the options object as the..." is just confusing to me. I would suggest changing this back.